nautobot/extras/tests/test_filters.py

```diff
@@ -1524,6 +1532,10 @@ def test_regression_distinct_2963(self):
             2,
         )
 
+    @skip("Skipping RelationshipAssociation is a through model.")
```

Comment:
I think it might be perfectly reasonable to expect through models to also support q filters - thoughts?

Reply:
What should be the filter predicate for `RelationshipAssociation` q filter?

Reply:
Mabye something like `relationship__label, relationship__key`?